### ISSUE TITLE
HammerCore mod Crash / Worlds won't load

### ISSUE THREAD

**Issue by spidyswan** [TerrariaCraft/Issues#71]
I just installed CurseForge and TerrariaCraft modpack on an OS X (10.15.7-Catalina) environment but I, so far ave been unable to get a world to load.

The first issue is that after tapping "PLAY" in the MC Launcher (which was launched from CurseForge) MC crashes and I get an error report that indicates Hammer Core as the problem. I have HammerLib-1.12.2-2.0.6.31 mod installed from the original modpack.

From there I can go back to the title screen and attempt to create a world, but the world is never created and stays at "0%"

Any clear and direct advice would be appreciated as I am fairly new to all of this.

Thanks.

<img width="600" alt="Screen Shot 2022-01-05 at 1 28 43 PM" src="https://user-images.githubusercontent.com/97190623/148273635-91494c20-857a-4ed5-ae3e-56b71a1d3ecc.png">
<img width="600" alt="Screen Shot 2022-01-05 at 1 35 01 PM" src="https://user-images.githubusercontent.com/97190623/148273645-63504ee6-8e3e-4aaa-8e24-f0146be6b096.png">

**Comment by Zeitheron** [comment]
Please provide the crash report, upload it to pastebin and post the URL here.

**Comment by spidyswan** [comment]
https://pastebin.pl/view/238a55d6

**Comment by Zeitheron** [comment]
Disable McCue, it is supported only within Windows environment

**Comment by spidyswan** [comment]
https://pastebin.pl/view/a5ff75c6

OK, this is the most recent crash report.

Sorry for this mess up, but the previous crash report was from yesterday. It shows Mc Cue as active. I have already removed MC Cue from the mods folders as it had caused an error previously. I just now removed a "mccue" folder form the "config" folder...no change. Still getting the same crash error for Hammer Core and no progress on loading a world.
